RogueMaster/flipperzero-firmware-wPlugins
# Restored Infrared and RFID CLI

# What's new

- Since the move to FAP for Infrared and RFID, CLI was missing - This restores that functionality

# Verification 

- Get into CLI
- Run the Infrared/RFID commands
- Ensure the commands are working properly

# Checklist (For Reviewer)
